src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-details/configuration-details.component.ts

```diff
@@ -3,16 +3,14 @@ import { Component, Input, OnChanges } from '@angular/core';
 import { I18n } from '@ngx-translate/i18n-polyfill';
 import * as _ from 'lodash';
 
-import { CdTableSelection } from '../../../../shared/models/cd-table-selection';
-
 @Component({
   selector: 'cd-configuration-details',
   templateUrl: './configuration-details.component.html',
   styleUrls: ['./configuration-details.component.scss']
 })
 export class ConfigurationDetailsComponent implements OnChanges {
   @Input()
-  selection: CdTableSelection;
+  selection: undefined;
   selectedItem: any;
```

Comment:
```suggestion
```

Reply:
Same here, I'd like to avaid removing `selectedItem: any;` because when `selection` changes the `selectedItems` get updated in the `ngOnChanges`. `selectedItems` is also used in the template. This is more a common refactoring I guess and should not be solved in this pr, because it's already too huge.